pepper-apis/src/test/resources/redisson-test-jcache.yaml

```diff
@@ -16,6 +16,7 @@ singleServerConfig:
   database: 0
   dnsMonitoringInterval: 5000
 threads: 16
-nettyThreads: 32
+nettyThreads: 64
+pingConnectionInterval: 3000
```

Comment:
Is there a difference putting this at the top-level or under `singleServerConfig`?

Reply:
I am not sure. I based the configuration the original configuration on what they have for examples in the repo:
https://github.com/redisson/redisson/wiki/2.-Configuration#262-single-instance-yaml-config-format

Reply:
Okay. Was just asking because the two files are different. If the `redisson-jcache.yaml.ctmpl` file above looks right to you, then let's ship it.